testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/security/callerprincipal/GetCallerPrincipalWithNoDefaultSecurityDomainTestCase.java

```diff
@@ -152,7 +148,6 @@ public void testUnauthenticatedNoSecurityDomain() throws Exception {
             assertEquals(ANONYMOUS, principal.getName());
         } catch (RuntimeException e) {
             e.printStackTrace();
-            log.error(e.getStackTrace());
```

Comment:
This could be better than e.printStackTrace(), WDYT?
```
StringWriter errors = new StringWriter();
e.printStackTrace(new PrintWriter(errors));
logger.error(errors.toString());
```